conventions/process.md

```diff
@@ -0,0 +1,52 @@
+# Process
+
+## Prioritisation
+
+Unless specifically raised the normal order of priority is:
+* The board of your product team comes before the component board.
+* The right-hand of the board comes before the left-hand side of the board.
+
+## I just finished a task, what do I do next
+Before starting a brand new task first help old task to be finished.
+Following the priority order check:
+
+### You product team board
+
+1) There is any task that you implemented that needs your attention (eg: update with the latest master, update the staging machine).
+
+2) There is any task that needs code review?
+
+### React board
+
+3) repeat 1 for the React board.
+4) repeat 2 for the React board.
+
+### Pick a new task
+
+5) Check the task in TODO for your product team board, from top to bottom pick the first unassigned.
+
+If there are **no unassigned** task in TODO in your product team board ask:
+
+* Other developers in your team if you can pick some task assigned to them.
+* The product manager of your team if some task on top of the backlog is ready to be worked with.
+* The Frontend Tech Lead (this is your last resource).
+
+
+## Picking up a new task
+
+(aka The three amigos meeting)
+
+When picking up a new task first thing to do is to verify that you have everything to work with that task.
+
```

Comment:
I would say this step should be done before estimations. Imagine a case, we agreed to complete X tasks during a sprint. When a person starts to work on a task, he suddenly finds out, that backend is not ready, or some other stuff is missing, than needs additional time. Thats were we finish with < 2 tasks done a week.

The way I see it. Everyone should spend 20 mins before sprint planning to get accustomed with tasks and see if there are any inconsistencies (BE from BE perspective, FE from EF and etc).

Then, during sprint planning we will be able to discuss things in more detail and use free amigos approach to deal with all issues before addressing estimations what tasks will be done during next sprint.

The way we did in my previous job is that we split tasks between team members and did 20min reviews each taking to look to 3/2 tasks.  So no need to you Marco to look at all tasks by yourself

Reply:
To address the size of the ticket we have the backlog grooming session (aka preplanning).

If you discuss every ticket in details during planning/preplanning you will have constantly half of the team idle during the meeting, since only a few people work at each ticket.

Three amigos can be done between preplanning and planning. But when you pick it up you should verify that everything is clear to all the people involved. 

Also that is the complete list of checks, hopefully, a good part should already be covered during the meetings, so when you pick the card there is very little to do.


In general, I think that the main point is what to do, and what is the deadline to do it (before start writing code). If you want to do that slightly differently and before is completely up to the implementer.